src/main/java/uk/gov/hmcts/reform/waworkflowapi/common/TaskToCreate.java

```diff
@@ -58,10 +68,11 @@ public int hashCode() {
     @Override
     public String toString() {
```

Comment:
Minor suggestion could be easier to use lombok here:
so 
@ EqualsAndHashcode
@ ToString